large_image/report.py

```diff
@@ -0,0 +1,65 @@
+import scooby
+
+
+class Report(scooby.Report):
+    def __init__(self, additional=None, ncol=3, text_width=80, sort=False):
+        """Initiate a scooby.Report instance."""
+        # Mandatory packages.
+        core = [
+            'large_image',
+            'cachetools',
+            'PIL',
+            'psutil',
+            'numpy',
+            'palettable',
+            'scooby',
+        ]
+        # Optional packages.
+        girder = [
+            'girder_large_image',
+            'girder_worker_utils'
+            'girder_worker'
+        ]
+        sources = [
+            'large_image_source_bioformats',
+            'bioformats',
+            'large_image_source_deepzoom',
+            'large_image_source_dummy',
+            'large_image_source_gdal',
+            'osgeo.gdal',
+            'pyproj',
+            'large_image_source_mapnik',
+            'mapnik',
+            'large_image_source_nd2',
+            'importlib',
+            'nd2reader',
+            'large_image_source_ometiff',
+            'large_image_source_openjpeg',
+            'glymur',
+            'large_image_source_openslide',
+            'openslide',
+            'large_image_source_pil',
+            'large_image_source_test',
+            'large_image_source_tiff',
+            'libtiff',
+            'large_image_converter',
+        ]
+        optional = [
+            'pylibmc',
+            'matplotlib',
+            'colorcet',
```

Comment:
Why do we mention packages we don't mention elsewhere in the project?  If you want to list all imported modules, rather than tracking them, why not just do something like `pprint.pprint({key: sys.modules[key].__version__ for key in sys.modules if hasattr(sys.modules[key], '__version__')})` -- though clearly you could add scooby's version detection logic for a better result.